gui/widgets/tab.cpp

```diff
@@ -122,6 +121,7 @@ int TabWidget::addTab(const Common::U32String &title, const Common::String &dial
 	newTab.title = title;
```

Comment:
we might want to add a bool withScroll argument to addTab, so you can create tabs without the automatic ScrollContainerWidget, or else we will have to fix all the old tabs that used scrollbars like GameOptions_KeyMapper

I do want to figure out why the nested scrollbars aren't working properly before I do either of those above things